config/environments/production.rb

```diff
@@ -124,10 +124,16 @@
 
   if ENV["RAILS_LOG_TO_STDOUT"].present?
     config.rails_semantic_logger.add_file_appender = false
+    config.rails_semantic_logger.filter = proc { |log| log.name != "DfE::Analytics::SendEvents" }
```

Comment:
I thought about skipping all via something like this but we'd probably miss out on DfE::Analytics errors too (which are in the format `DfE::Analytics::BigQueryApi::SendEventsError`).

```suggestion
    config.rails_semantic_logger.filter = proc { |log| log.name !~ /DfE::Analytics/ }
```